Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:7.4-cli
+FROM php:8.1-cli-buster
```

Comment:
proč ne rovnou na 8.2?

Reply:
Nechtělo se mi dělat matrix test na víc verzí a chtěl jsem mít jistotu, že na 8.1 to bude fungovat jak má. Ale klidně to dodělám 🤷‍♂️